### ISSUE TITLE
Interpretation of spatial effects if alpha = 0 for all spatial latent factors

### ISSUE THREAD

**Issue by jonpeake** [hmsc-r/HMSC#163]
As I understand it, a 0 alpha means that there is no spatial structure. However, in my case the posterior for the Eta and Lambda terms are still generated despite all spatial latent factors having an alpha equal to 0, resulting in a non-trivial variation partitioning for the spatial random effect. If spatial replicates are at the sample level (i.e., no duplicated coordinates among samples), how should we interpret the spatial term? Would they essentially be attributed to a non-spatial structured site-level random effect, and thus pooled together with the Random: Site term?

**Comment by ovaskain** [comment]
If the estimate of alpha equals zero, then the random effect indeed becomes non-spatial and hence the same as if one would set it originally through units=… rather than sData=…. So yes, if site is set both as spatial and unstructured, and if spatial is estimated as alpha=0, the same term is included essentially twice.

O2

From: Jonathan Peake ***@***.***>
Sent: torstai 27. heinäkuuta 2023 1:32
To: hmsc-r/HMSC ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [hmsc-r/HMSC] Interpretation of spatial effects if alpha = 0 for all spatial latent factors (Issue #163)


As I understand it, a 0 alpha means that there is no spatial structure. However, in my case the posterior for the Eta and Lambda terms are still generated despite all spatial latent factors having an alpha equal to 0, resulting in a non-trivial variation partitioning for the spatial random effect. If spatial replicates are at the sample level (i.e., no duplicated coordinates among samples), how should we interpret the spatial term? Would they essentially be attributed to a non-spatial structured site-level random effect, and thus pooled together with the Random: Site term?

**Comment by jonpeake** [comment]
Okay, thank you! So in this case, does it make more sense to eliminate the spatial term and re-run the model with just the unstructured term? Or can it be safely assumed that the sum of the spatial and unstructured terms would be equivalent to the output if only an unstructured term were used?

**Comment by ovaskain** [comment]
You can equally well drop either term, or use the sum (but the sum is a bit more complicated as the factors do not correspond to each other 1-1).

O2

**Comment by OceBartho** [comment]
Hi,
Quite similarly, in a probit HMSC model, I got all alpha = 0 for the spatial latent factor (defined as the coordinates of the sample sites). This random effect is included only as a spatially structured random effect. What I don't manage to understand, it is why if there should not be any spatial signal, this random effect still has in average a 10% explanatory power in the variance partitioning. 
What could cause such discrepancy?

Thank you in advance!

All the best,

Océane

**Comment by jonpeake** [comment]
> Hi, Quite similarly, in a probit HMSC model, I got all alpha = 0 for the spatial latent factor (defined as the coordinates of the sample sites). This random effect is included only as a spatially structured random effect. What I don't manage to understand, it is why if there should not be any spatial signal, this random effect still has in average a 10% explanatory power in the variance partitioning. What could cause such discrepancy?
> 
> Thank you in advance!
> 
> All the best,
> 
> Océane

From my understanding of the code, if alpha is 0, Eta is computed as if the factor was not spatially structured (see lines 77-81 [here](https://github.com/hmsc-r/HMSC/blob/master/R/computeDataParameters.R)) and thus so is Lambda (which is what's used in the variance partitioning).

**Comment by ovaskain** [comment]
Yes this is correct: if alpha is estimated to be zero, then the latent factor is simply non-spatial (the same that would have been obtained by setting units=… rather than sData=…). It can explain any amount of variation, so there is no discrepancy between alpha=0 and variance>0.

Best,

O2
